python/taichi/main.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, debug: bool = False, test_mode: bool = False):
     def __call__(self):
         # Print help if no command provided
         if len(sys.argv[1:2]) == 0:
-            self.main_parser.print_help()
+            self.help()
```

Comment:
I think this works in general, except `ti -h` won't print the links, which is fine to me.

Reply:
Yes, I just confirmed that `ti -h` doesn't print the links. Any idea on how to solve this?

Reply:
Solved by moving links to right-below of the banner.